test/message_bus/message_inbox/inbox_message_hash.js

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
```

Comment:
Please follow dev guideline for imports
https://github.com/mosaicdao/developer-guidelines/pull/1/files#diff-e31f7c3df082c6429fa66f20494f70bbR492

Check for other files too

Reply:
Changed the imports order.